libraries/AP_HAL_SITL/sitl_gps.cpp

```diff
@@ -426,7 +426,13 @@ void SITL_State::_update_gps_ubx(const struct gps_data *d, uint8_t instance)
         const Vector3f ant2_pos = _sitl->gps_pos_offset[instance].get();
         Vector3f rel_antenna_pos = ant2_pos - ant1_pos;
         Matrix3f rot;
+        // project attitude back using gyros to get antenna orientation at time of GPS sample
+        Vector3f gyro(radians(_sitl->state.rollRate),
```

Comment:
```suggestion
        Vector3f gyro{radians(_sitl->state.rollRate),
```

adds static checking, which is always nice